### ISSUE TITLE
iPad thermometers "jump" out of the way even with no temperature conflict

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-integers#91]
**Device**
iPad
**OS**
iPadOS 13
**Browser**
Safari
**Problem**
For https://github.com/phetsims/QA/issues/469
I noticed on iOS 13 (and maybe iOS 12) that, if you try to drop a thermometer on the climate map, it will do a bit of a "jump" as if you tried to drop it on a conflicting temperature even when there is none. Confirmed that @lmulhall-phet experienced this as well, so it is not me twitching as I let go. Does not seem to happen on Windows 10 or iOS 9. Does seem to happen in rc.3, rc.2, and rc.1. I don't think this should block publication but it probably should be looked at for an update when @jbphet comes back. @kathy-phet and @ariel-phet for their thoughts.

This may be hard to see in the gif
https://drive.google.com/file/d/1H8CuvEp5W8gRm0lPhEDeFctN-0uZlu5R/view?usp=sharing

**Comment by kathy-phet** [comment]
I agree - wouldn't block publication for this, but have JB take a look when he returns. Adding Amanda.

**Comment by ariel-phet** [comment]
Definitely not a showstopper for publication, worth @jbphet looking at upon his return.

**Comment by jbphet** [comment]
I looked at this for a while, and added `alert` calls (since I can't use `debugger` statements on an iPad without tethering it) to detect any post-release position changes, and I'm pretty convinced that this is just how touch handling works on the iPad.  I also added `alert` calls to detect if the code that repositions the thermometers in the event of overlapping points on the number line was firing, and it's not.

As a reality check, I tried positioning the items on the first scene very accurately, like placing the point of the bird's beak exactly at the top of one of the pieces of seaweed.  I saw the same jump on release.

Here is what I would suggest: Try it with a stylus, taking care to position it very carefully and release it straight up.  If it can still be duplicated under these conditions, I'll look at it some more.  If not, this should just be closed.

For reference, I'm testing on the PhET iPad Air 2 named "Pauling", iOS version 13.4.
